### ISSUE TITLE
Rack v1: Invalid tags in plugin.json

### ISSUE THREAD

**Issue by cschol** [hdavid/VCVRack-Holon.ist#8]
The manifest validator script found some invalid tags in your `plugin.json`:

```
Validating HolonicSystems-Free
SwissCheeseKnife: invalid tags: SAMPLE_AND_HOLD, SLEW_LIMITER_TAG
```

See https://github.com/VCVRack/Rack/blob/v1/src/plugin.cpp#L534 for an overview of valid tags.

**Comment by hdavid** [comment]
Hi @cschol,

thanks, i fixed the tags. new commit id : 0d4630044328b9f01dccc4abd5045ab4a229b461

**Comment by cschol** [comment]
This tag is still invalid:

```
Validating HolonicSystems-Free
SwissCheeseKnife: invalid tags: SLEW LIMITER TAG
```

**Comment by hdavid** [comment]
oups ! sorry about this.

fixed in this commit : d8519da26d114e2cfa811f9c1dd4cdbd36f06307
